lang/python/python-attrs/Makefile

```diff
@@ -8,16 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-attrs
-PKG_VERSION:=21.4.0
+PKG_VERSION:=23.1.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=attrs
-PKG_HASH:=626ba8234211db98e869df76230a137c4c40a12d72445c45d5f5b716f076e2fd
+PKG_HASH:=6279836d581513a26f1bf235f9acd333bc9115683f14f7e8fae46c98fc50e015
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Jeffery To <jeffery.to@gmail.com>
 
+PKG_BUILD_DEPENDS:=python-hatchling/host python-hatch-vcs/host python-hatch-fancy-pypi-readme/host
```

Comment:
I have one small nitpick. Would it be possible to describe such change in the commit description? Maybe, you can also add that you are assigning the copyright to yourself.

Reply:
I can mention in the commit message that the build dependencies have changed though I don't recall seeing this as a usual practice.

AFAIK I've never explicitly assigned copyright to OpenWrt so changing that line is correcting it

Reply:
Much better this time. Yeah, but it is also not a good idea to hide addding more build dependencies in the simple bump commit, so that's why I think, it was and it is still good idea to include it like you did. ;-)